service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEmailTemplateTest.java

```diff
@@ -0,0 +1,112 @@
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@SpringBootTest(classes = {
+    FurtherEvidenceUploadedEventHandler.class,
+    FurtherEvidenceUploadedEvent.class,
+    FurtherEvidenceNotificationService.class,
+    FurtherEvidenceUploadedEmailContentProvider.class,
+    LocalAuthorityEmailLookupConfiguration.class,
+    InboxLookupService.class,
+    NotificationService.class,
+    ObjectMapper.class,
+    CaseUrlService.class
+})
+class FurtherEvidenceUploadedEmailTemplateTest extends EmailTemplateTest {
+    private static final Long CASE_ID = 12345L;
+    private static final String LA_EMAIL = "la@example.com";
+    private static final String REP_EMAIL = "resp@example.com";
+    private static final LocalDateTime HEARING_DATE =
+        LocalDateTime.of(2021, 2, 22, 0, 0, 0).plusMonths(3);
+
+    @Autowired
+    FurtherEvidenceUploadedEventHandler underTest;
+
+    @Test
+    void sendNotification() {
+        UUID representativeUUID = UUID.randomUUID();
+
+        Representative representative = Representative
+            .builder()
+            .email(REP_EMAIL)
+            .role(RepresentativeRole.REPRESENTING_RESPONDENT_1)
+            .servingPreferences(RepresentativeServingPreferences.DIGITAL_SERVICE)
+            .build();
+
+        Respondent respondent = Respondent.builder()
+            .party(RespondentParty.builder().lastName("Smith").build())
+            .representedBy(wrapElements(List.of(representativeUUID)))
+            .build();
+
+        CaseData caseData = CaseData.builder()
+            .furtherEvidenceDocumentsLA(
+                wrapElements(SupportingEvidenceBundle.builder()
+                    .name("Non-Confidential Evidence Document 1")
+                    .uploadedBy("LA")
+                    .dateTimeUploaded(LocalDateTime.now())
+                    .document(DocumentReference.builder().build())
+                    .build()))
+            .familyManCaseNumber(CASE_ID.toString())
+            .representatives(List.of(element(representativeUUID, representative)))
+            .respondents1(wrapElements(respondent))
+            .id(CASE_ID)
+            .hearingDetails(wrapElements(HearingBooking.builder().startDate((HEARING_DATE)).build()))
+            .build();
+
+        CaseData caseDataBefore = CaseData.builder()
+            .id(CASE_ID)
+            .build();
+
+        underTest.handleDocumentUploadedEvent(
+            new FurtherEvidenceUploadedEvent(caseData, caseDataBefore, true,
+                UserDetails.builder().email(LA_EMAIL).forename("The").surname("Sender").build()));
+
+        assertThat(response())
+            .hasSubject("New documents uploaded, Smith")
+            .hasBody(emailContent()
+                .line("The Sender has uploaded evidence documents for:")
+                .line()
+                .line("^^Smith, 12345, hearing 22 May 2021")
```

Comment:
you can use .callout("Smith, 12345, hearing 22 May 2021")
right now it does not work, see comment in FurtherEvidenceUploadedEmailContentProvider